packages/core/src/index.ts

```diff
@@ -23,8 +25,6 @@ export {default as Head} from 'next/head'
 
 export {default as Link} from 'next/link'
 
-export {Router, useRouter, withRouter} from 'next/router'
```

Comment:
We still need to export `Router` from `next/router`

Reply:
Fixed.